features/createroom/impl/src/main/res/values-be/translations.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-  <string name="screen_create_room_action_create_room">"Новы пакой"</string>
-  <string name="screen_create_room_action_invite_people">"Запрасіце сяброў у Element"</string>
-  <string name="screen_create_room_add_people_title">"Запрасіць людзей"</string>
   <string name="screen_create_room_error_creating_room">"Пры стварэнні пакоя адбылася памылка"</string>
   <string name="screen_create_room_private_option_description">"Паведамленні ў гэтым пакоі зашыфраваны. Гэта шыфраванне нельга адключыць."</string>
   <string name="screen_create_room_private_option_title">"Прыватны пакой (толькі па запрашэнні)"</string>
   <string name="screen_create_room_public_option_description">"Паведамленні не зашыфраваны, і кожны можа іх прачытаць. Вы можаце ўключыць шыфраванне пазней."</string>
   <string name="screen_create_room_public_option_title">"Адкрыты пакой (для ўсіх)"</string>
   <string name="screen_create_room_topic_label">"Тэма (неабавязкова)"</string>
   <string name="screen_start_chat_error_starting_chat">"Пры спробе пачаць чат адбылася памылка"</string>
+  <string name="screen_create_room_action_create_room">"Новы пакой"</string>
+  <string name="screen_create_room_action_invite_people">"Запрасіце сяброў у Element"</string>
+  <string name="screen_create_room_add_people_title">"Запрасіць карыстальникаў"</string>
```

Comment:
I do not know why there is a reordering of keys, this is quite annoying. We may want to apply a post-treatment to order keys alphabetically...

Reply:
Localazy is supposed to order keys alphabetically, this is maybe a regression on their side?